src/fidesctl/core/apply.py

```diff
@@ -67,6 +67,34 @@ def echo_results(action: str, resource_type: str, resource_count: int) -> None:
     echo_green(f"{action.upper()} {resource_count} {resource_type} resource(s).")
 
 
+def validate_dataset_usage(taxonomy: Taxonomy) -> List:
```

Comment:
Nit, but this isn't super descriptive

```suggestion
def get_orphan_datasets(taxonomy: Taxonomy) -> List:
```